docs/networking/native-ipv6-networking.md

```diff
@@ -92,27 +95,48 @@ While default IPv6 addresses are configured automatically, you will need to stat
       auto lo
       iface lo inet loopback
 
-      # The primary network interface (IPv4)
+      # Automatically brings up the default network interface (eth0)
       auto eth0
-      iface eth0 inet dhcp
```

Comment:
Why have you removed this line, without re adding it? Will the Linode still get a DHCP-assigned IPv4 address after reboot without it?